en/advanced_config/ethernet_setup.md

```diff
@@ -0,0 +1,85 @@
+# PX4 Ethernet Setup
+
+Ethernet connectivity a faster, more reliable, and more flexible communication technology than serial communication approach.
+This method is preferred over serial communication in case of direct connection between Pixhawk and IP Radio telemetry, companion computers and Networks.
+It utilizes UDP connection for MAVLINK communication over the port 14550 with [MAVLINK2](https://mavlink.io/en/guide/mavlink_2.html). 
```

Comment:
Are you saying "by default PX4 will connect to the Ethernet on your specified address and stream MAVLink on port 14550?

1. That is the port for connecting a GCS which indicates that it probably streams the GCS profile, and that by default it is set up for connecting to a GCS. 
2. It may be that it also streams MAVLink for the companion profile to 14540 - if it does that then we'd state the default setup. Does it? If it does not then if you want to connect to a companion you will need to configure this.
3. So how do you configure the port in PX4 - is it just a serial port like any other from the PX4 perspective?

Reply:
1 and 2. It is broadcasting on 14550 port, I tested both with MAVSDK and also QGC. 
This commit says it. 
https://github.com/PX4/PX4-Autopilot/pull/18716/commits/70ca144c4d0257a9c7f2f5dda04b72d62a10b1b1
3. MAVLINK instance already starts with that port.

Reply:
@farhangnaderi No, that says just "1"

```
param set-default MAV_2_MODE 0  # `0` implies GCS. This would have to be `2` for MAVSDK or ROS
param set-default MAV_2_UDP_PRT 14550   # This port is for GCS. It might work with MAVSDK, but it is **not** for MAVSDK.
```

It is clear how you would set up "2", but not clear how you would set up `1` and `2`. I'll ask.

Reply:
Chasing this up in https://github.com/PX4/PX4-Autopilot/issues/19343

Note, the port setting above proves that PX4 just things Ethernet is another serial port. So we will need to update https://docs.px4.io/master/en/peripherals/mavlink_peripherals.html as well. The good news is that then we will be able to link to that for the docs here. Worth waiting on answer to the issue though.